execution/container.go

```diff
@@ -53,19 +53,19 @@ func NewContainer(ctx context.Context, stateDir, id, bundle string) (c *Containe
 	}()
 
 	if err = os.Mkdir(stateDir, 0700); err != nil {
-		err = errors.Wrap(err, "failed to create container state dir")
+		err = errors.Wrapf(err, "failed to create container state dir %q", stateDir)
```

Comment:
I think I didn't include those because the error returned by the `os` function usually already include the path, but I guess it doesn't hurt